src/components/Health/__tests__/__snapshots__/HealthDetails.test.tsx.snap

```diff
@@ -12,7 +12,7 @@ exports[`HealthDetails renders deployments failure 1`] = `
       size="sm"
       title={null}
     />
-     Error Rate over last 1m: 
+     Error Rate over 1m: 
```

Comment:
@jshaughn is it an intended change or not? Do you think it's fine to remove "last" here? I can understand when reading "Error rate over 1m" that it's implicitly the *last* 1 minute, but just want to make sure it's ok with you too.

Reply:
Good eye, I have no idea wht that "last" was omitted.  I'll add it back.

Reply:
Oh wait, this is a snapshot.  I must have just updated snapshots.  It took me a moment but now I see the unintended consequence of my change to the duration dropdown values.  If we are looking at live data then the "last" prefix makes sense but if we are looking at historical data it doesn't.  And health may very well [soon] be reporting historical health given the support we are adding for custom time ranges.  I think maybe RateIntervals.getName() should take a prefix  parameter, or just never add "last".   Let me know what you want and I can update it in the PR.

Reply:
It's fine for me to remove "last" here

Reply:
Well in the time since my last reply I noticed that thechange I made created a problem because MetricsDuration was depending on the text value in the serverConfig.durations.  The dropdpowns for metrics became just time vaues like. "1m", "5m".

Sooooo, in this latest revision I've mde the values consistent again, but instead of the very situational "Last XX", its now "XX Traffic".  This imo is a better value in general, as it is more descriptive of what the dropdown actually represents (the duration of traffic data per query).  UX is OK with the change, what do you think?

Reply:
Fine!